src/Form/Type/EmailTemplateType.php

```diff
@@ -69,4 +79,17 @@ public function getBlockPrefix(): string
     {
         return self::BLOCK_PREFIX;
     }
+
+    private function getAvailableEmailTemplateTypes(EmailTemplateInterface $emailTemplate): array
```

Comment:
Can you extract this into new service and spec it?

Reply:
I don't see specs for it

Reply:
Done